stork/doc/sqa_reports.yml.app

```diff
@@ -0,0 +1,8 @@
+Applications:
+    stork:
+        exe_directory: ${ROOT_DIR}
+        exe_name: stork
```

Comment:
```suggestion
```

These lines were not necessary in my Stork, since a base application will have its executable in the root directory of the code, and are removed when SQA init occurs anyway. Could you doublecheck that this is also the case in your tests?

Reply:
You are correct.